sdk/appconfiguration/app-configuration/test/internal/node/throttlingRetryPolicy.spec.ts

```diff
@@ -85,6 +85,9 @@ describe("Should not retry forever", () => {
       });
     } catch (error) {
       errorWasThrown = true;
+      if (!(error instanceof Error)) {
+        throw new Error("Error of unexpected kind is thrown");
+      }
```

Comment:
In situations like this where you catch something you can't handle, IMO it's best to just re-throw it.

```suggestion
      if (!(error instanceof Error)) {
        throw error;
      }
```